compose.override.production.yml

```diff
@@ -26,8 +26,9 @@ services:
       - production_postgres_data_backups:/backups:z
     env_file:
       - ./.envs/.production/.postgres
+      - /etc/vault.d/secrets/kv_root_security.env
```

Comment:
I was thinking that we'd just fix that as part of 88. Since we already do this in production with other services (django) it won't cause any issues so I figured that I'd wait until we actually came to test out and decide® on what we wanted to do for the env variablization stuff that way I don't start us down a path here that hasn't been fully thought-through.

Reply:
I think that's an ok plan. I was just confused to see this line added is all.

Reply:
Ah no that's fine, I just was confused to see this line added to the MR